.github/workflows/check_cirrus_cron.yml

```diff
@@ -17,4 +17,14 @@ jobs:
   # Ref: https://docs.github.com/en/actions/using-workflows/reusing-workflows
   call_cron_failures:
     uses: containers/podman/.github/workflows/check_cirrus_cron.yml@main
-    secrets: inherit
+    secrets:
+      SECRET_CIRRUS_API_KEY:
+        required : true
```

Comment:
I don’t think this should work? Compare the different documentation/structure of  https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#onworkflow_callsecretssecret_id vs. https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#onworkflow_callsecretssecret_id .

Or https://docs.github.com/en/actions/sharing-automations/reusing-workflows#passing-inputs-and-secrets-to-a-reusable-workflow .

Reply:
Ack, I will have another look tomorrow. A day more less won't change anything